### PR TITLE
Add: responsive admin page

### PR DESCRIPTION
**Pull request by drachno**

### CODE REVIEW

File: src/app/admin-page/stream-list/stream-list.component.ts [hunk 1]
```diff
@@ -22,7 +23,12 @@ export class StreamListComponent implements OnInit {
   streams$!: Observable<Stream[]>;
   streamForm!: FormGroup;
 
-  constructor(private streamService: StreamService, private store: Store, private fb: FormBuilder) { }
+  constructor(private streamService: StreamService,
```
Comment (GVabal) [inline src/app/admin-page/stream-list/stream-list.component.ts]: streamService nenaudojamas šitam komponente, ištrinčiau

**Review by GVabal — APPROVED** [review]

**Review by dziugas306 — APPROVED** [review]

File: src/app/admin-page/student-form/student-form.component.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
+<mat-dialog-content>
```
Comment (pauliusnet) [inline src/app/admin-page/student-form/student-form.component.html:2]: Just a suggestion:
```suggestion
<div mat-dialog-content>
```

should work the same or even solve the problem :)

File: src/app/admin-page/student-form/student-form.component.html [hunk 3]
```diff
@@ -96,4 +98,8 @@ <h1 *ngIf="isEditView">Student Edit Form</h1>
       </div>
     </div>
   </div>
-</form>
+  </form>
+</mat-dialog-content>
```
Comment (pauliusnet) [inline src/app/admin-page/student-form/student-form.component.html:102]:
```suggestion
</div>
```

File: src/app/admin-page/student-form/student-form.component.html [hunk 4]
```diff
@@ -96,4 +98,8 @@ <h1 *ngIf="isEditView">Student Edit Form</h1>
...
+<mat-dialog-actions>
```
Comment (pauliusnet) [inline src/app/admin-page/student-form/student-form.component.html:103]:
```suggestion
<div mat-dialog-actions>
```

**Pull request merged**